ospf6d/ospf6_top.c

```diff
@@ -428,6 +428,7 @@ static struct ospf6 *ospf6_create(const char *name)
 	o->ref_bandwidth = OSPF6_REFERENCE_BANDWIDTH;
 
 	o->distance_table = route_table_init();
+	o->distance_all = ZEBRA_OSPF6_DISTANCE_DEFAULT;
```

Comment:
This assignment changes the default distance that we send to zebra (look into `ospf6_distance_apply`).
Did you research how it affects the behavior?

Reply:
@idryzhov the reason why I changed the default value was because I wanted to print the administrative distance in `show ipv6 ospf` and the default value of 0 did not seem right to me. Zebra seems to change the default distance value to the correct value (110) at one point as seen in the routing table output. However it's true there may be side effects I cannot see... (I did not research beyond changing the distance value...) is it better to keep using 0 as default value and when printing it if it is 0 change it to 110 ? or ? thanks

Reply:
Sounds fine to me. Looks like a safer approach.

Reply:
When zebra receives a zero it is interpreted to mean that the distance will be changed to the distance of the protocol.  If ospfv3 passes down a non zero value then zebra respects that value( whatever it is ).  So having ospfv3 track and control the distance passed down is fine as well.  It doesn't matter a whole bunch